tests/inverse.js

```diff
@@ -33,4 +35,22 @@ QUnit.module('Тестируем функцию inverse', function () {
 		assert.deepEqual(inverse([ 1, 2, 3, 4, 5 ], -5), [ 1, 2, 3, 4, 5 ]);
 		assert.deepEqual(inverse([ 1, 2, 3, 4, 5 ], -15), [ 1, 2, 3, 4, 5 ]);
 	});
+
+/* Дополнительные тесты */
+
+	QUnit.test('Функция работает правильно с повторяющимися элементами', function (assert) {
+		assert.deepEqual(inverse([1, 1, 1, 1, 1]), [1, 1, 1, 1, 1]);
+		assert.deepEqual(inverse([1, 1, 1, 1, 2]), [2, 1, 1, 1, 1]);
+		assert.deepEqual(inverse([1, 1, 1, 2, 2]), [2, 2, 1, 1, 1]);
+	});
+
+	QUnit.test('Функция работает правильно с отрицательными элементами', function (assert) {
+		assert.deepEqual(inverse([ -1, -2, -3, -4, -5 ]),[ -5, -4, -3, -2, -1 ]);
+		assert.deepEqual(inverse([ -1, -2, -3, -3, -3 ]),[ -3, -3, -3, -2, -1 ]);
+
+	});
+
+	QUnit.test('Функция выдаёт исключения', function (assert) {
+		assert.throws(inverse([ 1, 2, 3, 4, 5 ], false), TypeError('Parameter is not a number!'));
```

Comment:
```suggestion
		assert.throws(() => inverse([ 1, 2, 3, 4, 5 ], false), TypeError('Parameter is not a number!'));
```
получается вместо того чтобы здесь вызывать и получать ошибку здесь - которую assert.throws перехватить не может, создается функа которая уже в try-catch вызывается внутри и делается проверка

а так - интернет всё знает https://www.google.com/search?q=js+assert.throws

Reply:
+ добавь тест на то что первый аргумент не массив